src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/confirm/containers/Confirm.tsx

```diff
@@ -115,7 +121,7 @@ const Confirm = (props: IConfirmProps) => {
       routeParams.partyId,
       routeParams.instanceGuid,
     );
-  // eslint-disable-next-line react-hooks/exhaustive-deps
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
```

Comment:
Any particular reason this react-hooks/exhaustive-deps can't be fixed? 🤔

Reply:
Nothing other than it requires a minor rewrite. I had a different PR where I fixed all of these, but some of the changes caused tests to fail. So I think we have to take the easy ones first (this PR) and revisit the other ones later. All the others have been commented out as well to reduce noise when developing.